lib/packages/docutils/rstgen.nim

```diff
@@ -1503,11 +1504,10 @@ proc rstToHtml*(s: string, options: RstParseOptions,
 
   const filen = "input"
   var d: RstGenerator
-  initRstGenerator(d, outHtml, config, filen, options, myFindFile,
-                   rst.defaultMsgHandler)
+  initRstGenerator(d, outHtml, config, filen, options, myFindFile, msgHandler)
   var dummyHasToc = false
-  var rst = rstParse(s, filen, line=LineRstInit, column=ColRstInit,
```

Comment:
why drop LineRstInit, ColRstInit here and elsewhere?

Reply:
This is just in `rstToHtml`. I realized that `ColRstInit=0` is not applied here because it's a standalone function, not part of compiler reporting system, so `ColOffset=1` will **not** be added here. It will be handled by `defaultMsgHandler` in rst.nim. So this function must start with `column=1` itself.

You can actually see that previously error messages in asserts in this test were like `input(6, 0)`, which was changed to `input(6, 1)`, etc.

Reply:
I still think it's a code smell that we have in some places `line=1, column=1` and in other places `line=LineRstInit, column=ColRstInit` with ColRstInit = 0, but this can be addressed in future work (refs https://github.com/timotheecour/Nim/issues/638)

Reply:
actually you are right, it's not a correct way.

I made it use `ColRstInit` everywhere again and add a new constant `ColRstOffset` wherever it's needed.